source/faq.txt

```diff
@@ -113,33 +113,31 @@ is on the other side of the planet and has a latency of 10000
 milliseconds, setting the ``connectTimeoutMS`` to anything lower will
 prevent the driver from ever connecting to that member.
 
-Should I Use "socketTimeoutMS" as a Way of Preventing Long-Running Operations From Slowing Down the Server?
--------------------------------------------------------------------------------------------------------------
+What Happens to My Operations if My Client Disconnects?
+-------------------------------------------------------
 
-No, you should **not** use ``socketTimeoutMS`` to end operations that
-may run for too long and slow down the application. Attempting to do so
-may not achieve the intended result.
+Starting in {+mdb-server+} version 4.2, the server terminates
+:manual:`certain operations </release-notes/4.2/#client-disconnection>` such as
+aggregations and find operations if the client disconnects before it
+successfully executes an operation.
 
-Closing the socket causes a reconnect of the driver's connection pool,
-introducing latency to any other queued up operations. Chronically slow
-operations will, therefore, cause a large number of reconnect requests,
-negatively impacting throughput and performance.
+Some operations, including all write operations, continue to run on the
+{+mdb-server+} even if the client disconnects. This behavior can cause data
+inconsistencies if your application retries the operation after failure.
```

Comment:
Suggestion:

I think, but am not sure, that this refers to the disconnect as the failure and not some other type of write failure. If so, it would be helpful to describe it as a disconnect since that may not be similar to a general failure.

```suggestion
inconsistencies if your application retries the operation after a disconnection.
```